File: SpEditText/src/main/java/com/sunhapper/x/spedit/mention/span/BreakableSpan.java
package com.sunhapper.x.spedit.mention.span;

import android.text.Spannable;

/**
 * Created by sunhapper on 2019/1/24 .
 * 标记可以从中间删除的span
 */
public interface BreakableSpan extends DataSpan {

    boolean isBreak(Spannable spannable);
}

File: SpEditText/src/main/java/com/sunhapper/x/spedit/mention/span/BreakableSpan.kt
package com.sunhapper.x.spedit.mention.span

import android.text.Spannable

/**
 * Created by sunhapper on 2019/1/24 .
 * 标记可以从中间删除的span
 * 与IntegratedSpan互斥，默认实现中IntegratedSpan优先级高于BreakableSpan
 */
interface BreakableSpan {

    fun isBreak(spannable: Spannable): Boolean
}
